feg/gateway/sbi/sbi.go

```diff
@@ -0,0 +1,63 @@
+//go:generate oapi-codegen -generate types,skip-prune -o ../../openapi/sbi_15.8/TS29122NpConfiguration/TS29122NpConfiguration.gen.go --import-mapping TS29122_CommonData.yaml:magma/feg/gateway/sbi/TS29122CommonData,TS29571_CommonData.yaml:magma/feg/gateway/sbi/TS29571CommonData TS29122_NpConfiguration.yaml
+//go:generate oapi-codegen -generate types,skip-prune -o ../../openapi/sbi_15.8/TS29519ExposureData/TS29519ExposureData.gen.go --import-mapping TS29518_Namf_Communication.yaml:magma/feg/gateway/sbi/TS29518NamfCommunication,TS29571_CommonData.yaml:magma/feg/gateway/sbi/TS29571CommonData,TS29518_Namf_EventExposure.yaml:magma/feg/gateway/sbi/TS29518NamfEventExposure TS29519_Exposure_Data.yaml
+//go:generate oapi-codegen -generate types,skip-prune -o ../../openapi/sbi_15.8/TS29519ApplicationData/TS29519ApplicationData.gen.go --import-mapping TS29551_Nnef_PFDmanagement.yaml:magma/feg/gateway/sbi/TS29551NnefPFDmanagement,TS29522_TrafficInfluence.yaml:magma/feg/gateway/sbi/TS29522TrafficInfluence,TS29571_CommonData.yaml:magma/feg/gateway/sbi/TS29571CommonData,TS29122_CommonData.yaml:magma/feg/gateway/sbi/TS29122CommonData,TS29514_Npcf_PolicyAuthorization.yaml:magma/feg/gateway/sbi/TS29514NpcfPolicyAuthorization TS29519_Application_Data.yaml
+//go:generate oapi-codegen -generate types,skip-prune -o ../../openapi/sbi_15.8/TS29518NamfMT/TS29518NamfMT.gen.go --import-mapping TS29571_CommonData.yaml:magma/feg/gateway/sbi/TS29571CommonData,TS29518_Namf_EventExposure.yaml:magma/feg/gateway/sbi/TS29518NamfEventExposure TS29518_Namf_MT.yaml
+//go:generate oapi-codegen -generate types,skip-prune -o ../../openapi/sbi_15.8/TS29122NIDD/TS29122NIDD.gen.go --import-mapping TS29122_CommonData.yaml:magma/feg/gateway/sbi/TS29122CommonData,TS29571_CommonData.yaml:magma/feg/gateway/sbi/TS29571CommonData TS29122_NIDD.yaml
+//go:generate oapi-codegen -generate types,skip-prune -o ../../openapi/sbi_15.8/TS29122ReportingNetworkStatus/TS29122ReportingNetworkStatus.gen.go --import-mapping TS29122_CommonData.yaml:magma/feg/gateway/sbi/TS29122CommonData,TS29571_CommonData.yaml:magma/feg/gateway/sbi/TS29571CommonData TS29122_ReportingNetworkStatus.yaml
+//go:generate oapi-codegen -generate types,skip-prune -o ../../openapi/sbi_15.8/TS29122GMDviaMBMSbyxMB/TS29122GMDviaMBMSbyxMB.gen.go --import-mapping TS29572_Nlmf_Location.yaml:magma/feg/gateway/sbi/TS29572NlmfLocation,TS29122_CommonData.yaml:magma/feg/gateway/sbi/TS29122CommonData,TS29571_CommonData.yaml:magma/feg/gateway/sbi/TS29571CommonData TS29122_GMDviaMBMSbyxMB.yaml
+//go:generate oapi-codegen -generate types,skip-prune -o ../../openapi/sbi_15.8/TS29510NnrfNFDiscovery/TS29510NnrfNFDiscovery.gen.go --import-mapping TS29510_Nnrf_NFManagement.yaml:magma/feg/gateway/sbi/TS29510NnrfNFManagement,TS29503_Nudm_SDM.yaml:magma/feg/gateway/sbi/TS29503NudmSDM,TS29571_CommonData.yaml:magma/feg/gateway/sbi/TS29571CommonData TS29510_Nnrf_NFDiscovery.yaml
```

Comment:
i think oapi-codegen can grab the specs from github directly. Wouldn't it make more sense to get them from there instead of having to maintain them in magma?

Would it be possible to use filters to generate any spec dropped at sbi folder?